vcpkg/users/buildsystems/cmake-integration.md

```diff
@@ -30,25 +52,13 @@ include_directories(${CATCH_INCLUDE_DIR})
 
 ## IDE Integration
 
-### Visual Studio Code (CMake Tools Extension)
+### Visual Studio / Visual Studio Code
 
-Adding the following to your workspace `settings.json` will make CMake Tools automatically use vcpkg for libraries:
+We recommend using [CMake Presets] in both Visual Studio and Visual Studio Code.
 
-```json
-{
-  "cmake.configureSettings": {
-    "CMAKE_TOOLCHAIN_FILE": "<vcpkg-root>/scripts/buildsystems/vcpkg.cmake"
-  }
-}
-```
-
-### Visual Studio
+Learn more about Visual Studio at [Configure and build with CMake Presets in Visual Studio](/cpp/build/cmake-presets-vs).
```

Comment:
Suggested reword: "Learn more at: <LINK>". No point saying Visual Studio twice when it's in the URL anyway.